sass/elements/_elements.scss

```diff
@@ -40,65 +40,67 @@ figure {
 }
 
 .callout {
-	@include full-width;
-	background: $color__background-screen;
-	border-bottom: 5px solid $red;
-	color: $white;
-	padding-top: $padding * 2;
-	padding-bottom: $padding * 2;
-
-	a {
-		color: $white;
-	}
-
-	a {
-		text-decoration: none;
-	}
-
-	.widget {
-		@include wrap();
-	}
-
-	.widget-title {
-		color: $cream;
-		margin: 0;
-	}
-
-	.textwidget {
-		@include fluid-units( 'font-size', 1.125, 1.62 );
-		font-family: $font__alt;
-		font-size: 1.62rem;
-		text-align: center;
-	}
-
-	@media screen and ($medium-down) {
-		text-align: center;
-
-		.textwidget div {
-			margin-top: $padding;
-		}
-	}
-
-	@media screen and ($medium-up) {
-
-		.widget {
-			align-items: center;
-			display: flex;
-			flex-wrap: wrap;
-		}
-
-		.widget-title,
-		.textwidget {
-			flex: 50%;
-		}
-
-		.textwidget {
-			align-content: center;
-			display: flex;
-			flex-wrap: wrap;
-			justify-content: space-around;
-		}
-	}
+  display: none;
+  visibility: hidden;
+	//@include full-width;
```

Comment:
what? why are we hiding this?

Reply:
Because clients are gonna client

Reply:
hm  okay... Can you please remove from the DOM. And then just remove this entire block of text. I don't leave commented-out code in my repo. It should just be removed.

Reply:
Took a different approach. See latest push.